skill-tree /src/main/java/com/RDS/skilltree/Skill/SkillsController.java

```diff
@@ -0,0 +1,69 @@
+package com.RDS.skilltree.Skill;
+
+import com.RDS.skilltree.Exceptions.NoEntityException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.coyote.Response;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLIntegrityConstraintViolationException;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@RestController
+@Slf4j
+@RequestMapping("/v1/skills")
+public class SkillsController {
+    private final SkillsService skillsService;
+    public SkillsController(SkillsService skillsService){
+        this.skillsService = skillsService;
+    }
+
+    @PostMapping("/")
+    public ResponseEntity<?> createSkill(@RequestBody(required = true) SkillDRO skillDRO){
```

Comment:
What does `?` mean? can we please define the return type?

Reply:
If we define a DTO object then incase of null that cannot be used.

Reply:
Should we be sending null in any API? Shouldn't our response object be consistent across all API?

Reply:
Can we discuss this once, I didn't get this one.